unison-cli/src/Unison/LSP/FileAnalysis.hs

```diff
@@ -123,106 +123,11 @@ checkFile doc = runMaybeT do
   let fileAnalysis = FileAnalysis {diagnostics = diagnosticRanges, codeActions = codeActionRanges, fileSummary, typeSignatureHints, ..}
   pure fileAnalysis
 
--- | If a symbol is a 'User' symbol, return (Just sym), otherwise return Nothing.
```

Comment:
This block was split off into the `UnisonFile.Summary` module in `parser-typechecker` so it's accessible to `debug.format`